.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: "0"
```

Comment:
I suggest keeping it to 50, if MR has more 50 commits it should be squashed then.

Reply:
Fixed with https://github.com/temporalio/helm-charts/pull/352/commits/5190b68e1972ada4cdb405e5f0bfbd03f801e0ca